contracts/libraries/Commands.sol

```diff
@@ -47,5 +47,8 @@ library Commands {
     uint256 constant X2Y2_1155 = 0x1b;
     uint256 constant FOUNDATION = 0x1c;
     uint256 constant SWEEP_ERC1155 = 0x1d;
-    // COMMAND_PLACEHOLDER = 0x1e to 0x3f (all unused)
+    uint256 constant SEAPORT_V2 = 0x1e;
```

Comment:
given seaport is our most used NFT marketplace I think we should maybe change this to be 0x21 (I'm about to merge a PR that takes 0x20). That way it has minimal `if` checks before executing